templates/Users/register.php

```diff
@@ -35,6 +35,6 @@
         }
         ?>
     </fieldset>
-    <?= $this->Form->button(__d('cake_d_c/users', 'Submit')) ?>
+    <?= $this->User->button(__d('cake_d_c/users', 'Submit')) ?>
```

Comment:
why this change?

Reply:
You can see https://github.com/CakeDC/users/pull/1085/files#diff-627db50119981d9f17e65e3606d2a3e45c1292417ee73a2a88b5e5ed07cc9cceR214. Recaptcha V3 requires some properties to be added to the button